resources/execute_event_handler.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+resource_name :execute_event_handler
+provides :execute_event_handler
+
+# Resource to execute BYOS event handler
+
+property :event_name, String, name_property: true
+property :event_command, String, required: false
+
+default_action :run
+
+action :run do
+  if new_resource.event_command.nil?
+    Chef::Log.info("No command defined for Event #{new_resource.event_name}, noop")
+    return
+  end
+
+  event_log = '/var/log/parallelcluster/byos-plugin.log'
+  event_cwd = '/home/byos'
+  event_user = 'byos'
+  event_env = build_env
+  event_log_prefix_error = "[%Y-%m-%d %H:%M:%S] ERROR [#{new_resource.event_name}]"
+  event_log_prefix_info = "[%Y-%m-%d %H:%M:%S] INFO [#{new_resource.event_name}]"
```

Comment:
shall we use the ISO format for the timestamp here?

Reply:
nit shall the prefix be: `[date] [event] INFO: `?

Reply:
done both